### PR TITLE
EARTH-1646: updating all the references to Starter

### PR DESCRIPTION
**Pull request by jenbreese**
# READY FOR REVIEW

# Summary
- SDSS subtheme. Updated all the references to the starter kit.

# Review By (Date)
- soon

# Criticality
- Normal

# Urgency
- Normal

# Review Tasks

## Setup tasks and/or behavior to test

1. Check out this branch
2. Rebuild Cache and import config `drush cr ; drush ci`
3. Navigate to the appearances tab
4. Verify you can install the subtheme.
5. Verify no more reference to Starter.

### Site Configuration Sync

- Is there a config:export in this PR that changes the config sync directory?

## Front End Validation
- [ ] Design is approved by @ user?
- [ ] HTML validation: Is the markup using the appropriate semantic tags and [passes validation](https://validator.w3.org/nu/)? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Cross-browser testing: Has been performed? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Automated accessibility: Scans performed? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Manual accessibility: Manually tested? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?

### Code
- [ ] Are the naming conventions following our standards?
- [ ] Does the code have sufficient inline comments?
- [ ] Is there anything in this code that would be hidden or hard to discover through the UI?
- [ ] Are there any [code smells](https://blog.codinghorror.com/code-smells/)?
- [ ] Are tests provided? eg (unit, behat, or codeception)

### Code security
- [ ] Are all [forms properly sanitized](https://www.drupal.org/docs/8/security/drupal-8-sanitizing-output)?
- [ ] Any obvious [security flaws or new areas for attack](https://www.drupal.org/docs/8/security)?

## General
- [ ] Is there anything included in this PR that is not related to the problem it is trying to solve?
- [ ] Is the approach to the problem appropriate?

# Affected Projects or Products
- SDSS subtheme only

# Associated Issues and/or People
- [Earth-1646](https://stanfordits.atlassian.net/browse/EARTH-1646)

# Resources
- [AMP Tool](https://stanford.levelaccess.net/index.php)
- [Accessibility Manual Test Script](https://docs.google.com/document/d/1ZXJ9RIUNXsS674ow9j3qJ2g1OAkCjmqMXl0Gs8XHEPQ/edit?usp=sharing)
- [HTML Validator](https://validator.w3.org/)
- [Browserstack](https://live.browserstack.com/dashboard) and link to [Browserstack Credentials](https://asconfluence.stanford.edu/confluence/display/SWS/External+Account+Credentials)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
-See all the colors already available through decanter: https://decanter.stanford.edu/page/brand-design-elements-color/ 
+2. Fork or download this theme repository.
+3. Change all theme file names from including "sdss_subtheme" to including the machine name of your theme.
+4. Run a search and replace throughout the theme files to replace "sdss_subtheme" with the machine name of your theme.
+5. Add any specific brand colors you need (in addition to the decanter colors that are already available) to src/scss/utilities/variables/_colors.scss.
+See all the colors already available through decanter: https://decanter.stanford.edu/page/brand-design-elements-color/
 6. If desired, add font settings if you need to override and use fonts other than decanter fonts ( https://decanter.stanford.edu/page/brand-design-elements-typography/ ) [link],
 by defining a font library in the themename.libraries.yml file.
 7. If desired, add button mixins in src/scss/utilities/mixins/_buttons.scss with your button styles and then reference them and use them in src/scss/theme/_button.scss.
```
Comment (joegl) [inline README.md]: Should we leave this installation section in here? These look like instructions for the starter theme.

Comment (jenbreese) [reply README.md]: Good point. I'll pull them out.

Comment (jenbreese) [reply README.md]: I left in the colors and fonts line since that seems to make sense to leave. I can pull out if you have strong opinions on that one.

Comment (joegl) [reply README.md]: No I like that, it's still useful information.

**Review by joegl — APPROVED** [review]
LGTM.

**Pull request merged**